dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,291 @@
+      // .addPanel(
+      //     g.panel('Rebalance-Files') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Size') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Skipped') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      //   .addPanel(
+      //     g.panel('Rebalance-Failed') +
+      //     statPanelArgs +
+      //     g.statPanel('')
+      //   )
+      // The below variables needs to chanaged when cluster_id is introduced in the metrics
+      // local queryLabels = "{job='$OCS_Cluster', volume='$Volume'}";
+      // local fopQueryLabel = "{job='$OCS_Cluster', volume='$Volume', fop!=''}";
+      // Below variables are defined considering that there is only one gluster cluster.
+      local queryLabels = "{volume='$Volume'}";
+      local fopQueryLabel = "{volume='$Volume', fop!=''}";
+      local fopLockQueryLabel = "{volume='$Volume',fop='LOCK_OPS'}";
+      local fopInodeQueryLabel = "{volume='$Volume',fop='INODE_OPS'}";
+      local fopEntryQueryLabel = "{volume='$Volume',fop='ENTRY_OPS'}";
+      local fopReadWriteQueryLabel = "{volume='$Volume',fop='READ_WRITE_OPS'}";
+
+      // g.dashboard(
+      //   'K8s / Storage Resources / Persistent Volume / glusterfs',
+      //   uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      // ).addTemplate('OCS_Cluster', 'gluster_volume_up', 'job')
+      // .addTemplate('Volume', 'gluster_volume_up{job = "$OCS_Cluster"}', 'volume')
+      g.dashboard(
+        'K8s / Storage Resources / Persistent Volume / glusterfs',
+        uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
+      )
+      .addTemplate('Volume', 'gluster_volume_up', 'volume')
+      .addRow(
+        (g.row('Volume Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+        .addPanel(
+          g.panel('Status') +
+          statPanelArgs +
+          g.statPanel('gluster_volume_up{volume = "$Volume"}',) +
+          {
+            valueMaps: [
+              {
+                op: '=',
+                text: 'DOWN',
+                value: '0',
+              },
+              {
+                value: '1',
+                op: '=',
+                text: 'UP',
+              },
+            ],
+            thresholds: '0.9,1.1',
+            colors: [
+              '#cc0000',
+              '#92d400',
+              '#FFFFFF',
+            ],
+            colorValue: true,
+          }
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_used_bytes_total:sum%s / gluster:volume_capacity_total_bytes:sum%s' % [queryLabels, queryLabels])
```

Comment:
Is this percent? If so don't we need to multiply the result by 100?

Reply:
I agree. I am not able to see % sign added as per the mockup

Reply:
yes, this is percentage. Single stat by default takes it as percentage and multiplies by 100. Already taken care there in library.